science-of-connection.html

```diff
@@ -34,22 +34,57 @@
 
 		<section class="explanation-text-section">
 			<div class="section" >
-				<div class="container clearfix">
-					<div class="heading-block center">
-						<span class="non-emphasized-header">Neuroscience of </span>
-						<span class="emphasized-header">
-							Connection and connectedness:
-						</span>
-						
-					</div>
-					<div class="row">
-						<img src="images/owlScienceCrop.png" width="150px" height="auto" class="pull-right" />
-						<div class="col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-2 col-xs-12 col-xs-offset-2 text-center">
-							
-							<p>What does disconnectedness look like socially and behaviorally, and how does it affect the human brain? How can modern science support the concept of a spiritual and social illness?</p>
-							<p>When human beings are spiritually and socially sick/injured, and we *heal* by connection to a *transcendent nexus*: our personal experience of higher power/nature/source of consciousness. It connotes something that is1) Greater than self, and 2) Exists in a state of connection and interdependence to all elements of an indicidual’s environment. Illness of both the body and mind indicate disconnection and disintegration physically, psychologically, and socially-- from ones self and from one's existential nexus.  </p>
-						</div>
-					</div>
+				<div class="container clearfix text-center">
+          <img src="images/owlScienceCrop.png" width="150px" height="auto" class="bottommargin" />
+          <div class="heading-block center">
+            <span class="non-emphasized-header">Neuroscience of </span>
+            <span class="emphasized-header">
+              Connection and connectedness:
+            </span>
+            
+          </div>
+          <div class="row">
+            <div class="col-md-8 col-md-offset-2 bottommargin text-center">
+              <div class="team team-list clearfix">
+                <div class="team-image">
+                  <img src="images/rachel-headshot.jpg" alt="Rachel Wurzman, PhD, Chief Science Officer">
+                </div>
+                <div class="team-desc">
+                  <div class="team-title"><h4>Rachel Wurzman, PhD</h4><span>Chief Science Officer </span></div>
+                  <div class="team-content">
+                    <p>Background: Neuroscience PhD, post-doctoral research fellow in the LCNS at the University of Pennsylvania</p>
+                    <p>R&R: development and publication of academic research exploring the neuroscience of addiction behaviors& patterns.</p>
```

Comment:
Do you mind turning this into an HTML entity? And check the spacing toward the end. Thanks!

Reply:
No worries at all, I converted all the Ampersands now in 87ad113.

@j00bar ready for round 2 of review.